a11y/components/InputTextSpec.scala

```diff
@@ -72,7 +73,7 @@ class InputTextSpec extends A11ySpecBase {
           component
             .apply(
               form("value"),
-              TextInputWithHiddenLabel(title, label, additionalHtml),
+              TextInputWithHiddenLabel(title, caption, additionalHtml),
```

Comment:
This was formatted this way to avoid an accessibility issue. I think we need to close this PR.